server/package.json

```diff
@@ -17,7 +17,12 @@
         "vscode-languageserver-textdocument": "^1.0.4",
         "vscode-uri": "3.0.8"
     },
+    "devDependencies": {
+        "@types/mocha": "^10.0.0",
+        "ts-mocha": "^10.0.0"
```

Comment:
why decided on using these?

Reply:
Because vscode-test used for integration tests [uses mocha](https://code.visualstudio.com/api/working-with-extensions/testing-extension#quick-setup-the-test-cli). Decided that it would be better not to have different frameworks in one repo.

Reply:
ok I have no problem with that, just wanted to know the reasoning behind the decision